File: codigos/infos/Teste.java
public class Teste{
    static public void main(String[] args){
        String str = "oscar";
        int soma = 0;

        for(int i = 0; i < str.length(); i++){
            soma += (int)str.charAt(i);
        }

        System.out.println(soma);
    }
}

File: codigos/infos/teste.c
#include <stdio.h>

int main(){
    char str[] = "oscar";
    int soma = 0;
    
    for(int i = 0; str[i] != '\0'; i++){
        soma += (int)str[i];
    }

    printf("%d\n", soma);

    printf("%d", (int)str % 15);
}